.github/workflows/linux.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Install dependencies
         run:  sudo apt-get install -y $(./scripts/list-build-dependencies.sh -p apt)
       - name: Build
-        run:  ./scripts/build.sh
+        run: ./scripts/build.sh --release small --lto
```

Comment:
We don't need to optimize for small builds, it only slows down the CI. Default GCC optimizes for compilation speed and this is the only thing we care about for this CI job.